src/v/config/configuration.cc

```diff
@@ -3356,12 +3356,12 @@ configuration::configuration()
   , iceberg_enabled(
       *this,
       "iceberg_enabled",
-      "When set to true, enables the feature that translates topic data to "
-      "iceberg tables. This configuration only enables the feature at the "
-      "cluster level and requires individual topics to explicitly opt in by "
-      "setting the iceberg.enabled property to true. When the configuration "
-      "is set to false, disables the feature at the cluster level effectively "
-      "turning it off for all topics.",
+      "When set to true, this enables the feature that translates topic data "
+      "to Iceberg tables. This enables it at the cluster level, and it "
```

Comment:
> This enables it at the cluster level

This gives an impression that the cluster level default is set to true, should we reword it to say that this only enables the feature and the default for all topics is still false?

Reply:
Thanks @bharathv. How is this: 

Enables the feature that translates topic data to Iceberg tables. When set to true, this enables the feature at the cluster level and requires individual topics to explicitly opt in by setting the iceberg.enabled property to true. When set to false, this disables the feature at the cluster level, effectively turning it off for all topics.

Reply:
lgtm.